src/main/java/netcracker/study/controller/Bootstrap.java

```diff
@@ -0,0 +1,84 @@
+package netcracker.study.controller;
+
+import lombok.Getter;
+import lombok.Setter;
+import netcracker.study.api.controller.IBootstrap;
+import netcracker.study.api.repository.IProjectRepository;
+import netcracker.study.api.repository.ITaskRepository;
+import netcracker.study.api.service.IProjectService;
+import netcracker.study.api.service.ITaskService;
+import netcracker.study.command.AbstractCommand;
+import netcracker.study.error.NoSuchCommandsException;
+import netcracker.study.repository.ProjectRepository;
+import netcracker.study.repository.TaskRepository;
+import netcracker.study.service.ProjectService;
+import netcracker.study.service.TaskService;
+import org.reflections.Reflections;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+
+@Getter
+@Setter
+public class Bootstrap implements IBootstrap {
+
+    private final IProjectRepository projectRepository = new ProjectRepository();
```

Comment:
export field initialization into constructor

Reply:
only for this field?

Reply:
all of them and in every class